src/app/components/server/Icons.tsx

```diff
@@ -510,9 +510,6 @@ export const AvatarIcon = (
       />
       <mask
         id="mask0_328_1696"
-        style={{
-          maskType: "luminance",
```

Comment:
[This is the default value](https://developer.mozilla.org/en-US/docs/Web/CSS/mask-type#formal_definition), so removing this doesn't change anything. (It's the avatar icon at the bottom of the manual resolution page in the guided flow.)